histomicstk/segmentation/label/trace_boundary.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+import ctypes
+
+from histomicstk.segmentation.label.isbf import isbf
```

Comment:
@cdeepakroy got an import error when using the relative. Could you see [this error](https://travis-ci.org/DigitalSlideArchive/HistomicsTK/builds/161689528#L2722)